### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amontoison**
To see what looks like the new README.md
-> https://github.com/amontoison/Krylov.jl/tree/update-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-[![DOI](https://zenodo.org/badge/31977760.svg)](https://zenodo.org/badge/latestdoi/31977760)
-[![Build Status](https://travis-ci.org/JuliaSmoothOptimizers/Krylov.jl.svg?branch=master)](https://travis-ci.org/JuliaSmoothOptimizers/Krylov.jl)
-[![Build status](https://ci.appveyor.com/api/projects/status/3xt558lune9f5r2v?svg=true)](https://ci.appveyor.com/project/dpo/krylov-jl)
-[![Build Status](https://api.cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl.svg)](https://cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl)
-[![Coverage Status](https://coveralls.io/repos/github/JuliaSmoothOptimizers/Krylov.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaSmoothOptimizers/Krylov.jl?branch=master)
-[![codecov.io](https://codecov.io/github/JuliaSmoothOptimizers/Krylov.jl/coverage.svg?branch=master)](https://codecov.io/github/JuliaSmoothOptimizers/Krylov.jl?branch=master)
-[![](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaSmoothOptimizers.github.io/Krylov.jl/dev)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3548984.svg)](https://doi.org/10.5281/zenodo.3548984)
+
+| **Documentation** | **Travis, AppVeyor and Cirrus build Statuses** | **Coverage** |
```
Comment (dpo) [inline README.md]: statuses (small s)

File: README.md [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
+adjoint systems
+<p align="center">
+  <b><i>Ax = b</i></b>  and  <b><i>Aᵀy = c</i></b>,
+</p>
+saddle-point and SQD systems
+<p align="center">
+  [<b><i>M </i></b>&nbsp;&nbsp;&nbsp;<b><i> A</i></b>]&nbsp; [<b><i>x</i></b>]            =           [<b><i>b</i></b>].
+  <br>
+  [<b><i>Aᵀ</i></b>&nbsp;&nbsp;      <b><i>-N</i></b>]&nbsp; [<b><i>y</i></b>]&nbsp;&nbsp;&nbsp;&nbsp;[<b><i>c</i></b>]&nbsp; 
+</p>.
```
Comment (dpo) [inline README.md]: This looks good but it's redundant with the "Objective" section below. I think the two should be merged.

File: docs/src/index.md [hunk 3]
```diff
@@ -8,11 +8,18 @@ linear least-squares problems
...
-  \min \|x\| \quad \text{subject to} \ Ax = b.
+  \min \|x\| \quad \text{subject to} \quad Ax = b,
+```
+adjoint systems
+```math
+  Ax = b \quad \text{and} \quad A^T y = c,
+```
+saddle-point and SQD systems
+```math
+  \begin{bmatrix} M & \phantom{-}A \\ A^T & -N \end{bmatrix} \begin{bmatrix} x \\ y \end{bmatrix} = \left(\begin{bmatrix} b \\ 0 \end{bmatrix},\begin{bmatrix} 0 \\ c \end{bmatrix},\begin{bmatrix} b \\ c \end{bmatrix}\right).
```
Comment (dpo) [inline docs/src/index.md]: Same here.

**Comment by amontoison** [comment]
Thanks, I will merge both section.
And maybe we can also remove  `Long-Term Goals` section now?

**Comment by dpo** [comment]
> And maybe we can also remove Long-Term Goals section now?

Yes indeed.

File: README.md [hunk 4]
```diff
@@ -1,65 +1,80 @@
...
 <p align="center">
-  minimize ‖<b><i>b</i></b> - <b><i>Ax</i></b>‖.
+  minimize ‖<b><i>x</i></b>‖ &nbsp; subject to &nbsp; <b><i>Ax = b</i></b>
 </p>
 
-It is appropriate, in particular, in situations where such a problem must be solved but a factorization is not possible, either because:
+sould be solved when **_A_** is column rank-deficient but **_b_** is in the range of **_A_**.
+This situation occurs when **_b_** is in the range of **_A_** and
+   * **_A_** is square but singular,
+   * **_A_** is short and wide.
```
Comment (dpo) [inline README.md]: *A* could also be tall here, as long as *Ax=b* is consistent.

File: README.md [hunk 5]
```diff
@@ -1,65 +1,80 @@
...
+
+5. saddle-point or symmetric quasi-definite (SQD) systems
+
+<p align="center">
+  [<b><i>M </i></b>&nbsp;&nbsp;&nbsp;<b><i> A</i></b>]&nbsp; [<b><i>x</i></b>]            =           [<b><i>b</i></b>]
+  <br>
+  [<b><i>Aᵀ</i></b>&nbsp;&nbsp;      <b><i>-N</i></b>]&nbsp; [<b><i>y</i></b>]&nbsp;&nbsp;&nbsp;&nbsp;[<b><i>c</i></b>]
+</p>
+
+are also encountered in more specific applications. Krylov solvers are particularly appropriate in situations where such problems must be solved but a factorization is not possible, either because:
```
Comment (dpo) [inline README.md]: We need a new paragraph before "Krylov solvers...". Otherwise, it sounds like that sentence only applies to SQD systems.

**Comment by dpo** [comment]
Bump

**Comment by amontoison** [comment]
Updated. Thank you for the message, I forgot to push the modifications.

**Comment by dpo** [comment]
I think we could still make a number of changes here. Could you [give me write access](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to your branch?

**Comment by amontoison** [comment]
Yes, you should have write access now.

**Comment by dpo** [comment]
I made some changes. Let me know what you think.

**Comment by amontoison** [comment]
It's great! I did the same modifications in the documentation.

**Comment by dpo** [comment]
We should allow failures in the nightly version on FreeBSD.

**Comment by dpo** [comment]
Thank you.

**Pull request merged**